### ISSUE TITLE
Bad performance when swipe left/right to change month.

### ISSUE THREAD

**Issue by vinhtran6921** [stephy/CalendarPicker#179]
When I swipe left/ right to change month, it's very slow and sometime it doesn't change.
I tried on both IOS and android and had the same problem
How can I fix or improve it ?
Thanks for your help. This library is amazing

**Comment by peacechen** [comment]
Is that on a physical device or the emulator?  Is performance also bad when clicking on the left/right arrows to change months?

Swiping (and gestures in general) is difficult to get right with React Native.  If you have time to debug this, please submit a PR.

**Comment by rajubushanam** [comment]
Yes the library is amazing, I see the same issue on my phone when testing on Android.

**Comment by peacechen** [comment]
[React Native Gesture Handler](https://software-mansion.github.io/react-native-gesture-handler/) might improve swipe performance.  Please try refactoring `Swiper.js` to use that library and submit a PR if it's better.

**Comment by rajubushanam** [comment]
@peacechen Cool I will look into that and see if I can make it work.

**Comment by MannySAD** [comment]
@peacechen I have the same issue on my android device (have not tried it on iphone)... Any update on this enhancement?
@rajubushanam Did you make it work?
I just implemented this library. It has helped alot.

**Comment by peacechen** [comment]
@MannySAD 
This is a community supported library.  Your contributions will help yourself as well as others.  I have spent countless hours putting work into this project without compensation.  I'm not complaining -- the experience has been valuable, but I have limited time to work on all the requested features and bug fixes.

A guide on how to contribute to this and other open source libraries:
https://codeburst.io/a-step-by-step-guide-to-making-your-first-github-contribution-5302260a2940
